src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -53,12 +57,366 @@ uint32_t SystemRequest::index = 0;
 const std::string kSYNC = "SYNC";
 const std::string kIVSU = "IVSU";
 
+#define QUERY_APPS_VALIDATION_FAILED ":QUERY_APPS_VALIDATION_FAILED:"
+
+struct comparer {
```

Comment:
@okoreniak  Please reread googcle coding style. Types should started from capital letter